src/ReshowcaseUi.re

```diff
@@ -867,7 +838,7 @@ module DemoUnit = {
 
     <div name="DemoUnit" style=Styles.container>
       <div style=Styles.contents> {demoUnit(props)} </div>
-      {switch (parentWindowRightSidebarElem) {
+      {switch (sidebarElem) {
```

Comment:
I also refactored the sidebar to use reference instead of getting elements by id, which seems much more fragile and caused runtime exceptions for me.